aiven/resource_account_team_member.go

```diff
@@ -0,0 +1,223 @@
+package aiven
+
+import (
+	"fmt"
+	"github.com/aiven/aiven-go-client"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+var aivenAccountTeamMemberSchema = map[string]*schema.Schema{
+	"account_id": {
+		Type:        schema.TypeString,
+		Description: "Account id",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"team_id": {
+		Type:        schema.TypeString,
+		Description: "Account team id",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"user_email": {
+		Type:        schema.TypeString,
+		Description: "Team invite user email",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"invited_by_user_email": {
+		Type:        schema.TypeString,
+		Description: "Team invited by user email",
+		Optional:    true,
+		Computed:    true,
+	},
+	"create_time": {
+		Type:        schema.TypeString,
+		Description: "Time of creation",
+		Optional:    true,
+		Computed:    true,
+	},
```

Comment:
I would add `accepted` property like we have for project user resource for consistency.

Reply:
This conversation was marked as resolved but I don't see the value